.gitignore

```diff
@@ -16,3 +16,22 @@ libtool
 *.bak
 *.tmp
 src/dsql/parse.cpp
+
+# Visual Studio 2015 cache/options directory
+.vs/
+
+# Visual C++ cache files
+ipch/
+*.aps
+*.ncb
+*.opendb
+*.opensdf
+*.sdf
+*.cachefile
+
+# Visual Studio profiler
+*.psess
+*.vsp
+*.vspx
+*.sap
+
```

Comment:
Does MSVC really store all these files in the root directory of the source tree, not inside /builds/win32? We already have /builds/win32/.gitignore with:

ipch/
*.log
*.opensdf
*.sdf
*.suo
*.user
*.y